lib/elliptic/utils.js

```diff
@@ -37,7 +38,6 @@ function toArray(msg, enc) {
   }
   return res;
 }
-utils.toArray = toArray;
```

Comment:
Yeah, I can't even check if you exported everything without checking every name.